anap1003/assignment1/Q2.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment1;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+
+/**
+ *
+ * @author Ana
+ */
+public class Q2 {
+
+    private static int q2(int k, LinkedList list) {

```

Comment:
Again, use descriptive names for your methods. The user has no idea what q2 is doing.